ftsbot/cogs/textcommands.py

```diff
@@ -290,6 +290,45 @@ async def adminrequestrequirements(
 			)
 		)
 
+	@app_commands.command(
+		description='Inform someone as to why they could be getting Rate Limited!'
+	)
+	async def ratelimited(
+		self,
+		interaction: discord.Interaction
+	):
+		await interaction.response.send_message(
+			embed=discord.Embed(
+				colour=discord.Colour(0x00ffff),
+				title=('What actions could result in being Rate Limited?'),
```

Comment:
I don't know about the casing, I would probably just lowercase the "rate limited"